src/OpenTelemetry.Api/.publicApi/netstandard2.0/PublicAPI.Shipped.txt

```diff
@@ -108,6 +108,7 @@ OpenTelemetry.Trace.TelemetrySpan.Dispose() -> void
 OpenTelemetry.Trace.TelemetrySpan.End() -> void
 OpenTelemetry.Trace.TelemetrySpan.End(System.DateTimeOffset endTimestamp) -> void
 OpenTelemetry.Trace.TelemetrySpan.IsRecording.get -> bool
+OpenTelemetry.Trace.TelemetrySpan.ParentId.get -> System.Diagnostics.ActivitySpanId
```

Comment:
This change should go into the "Unshipped" file, not the "Shipped" file.

Reply:
Thought I'd removed it. Done.